lib/shoulda/matchers/active_record/have_rich_text_matcher.rb

```diff
@@ -67,7 +69,7 @@ def has_attribute?
         end
 
         def has_expected_action_text?
-          @subject.send(rich_text_attribute).class.name == 'ActionText::RichText'
+          @subject.send(rich_text_attribute).instance_of?(ActionText::RichText)
```

Comment:
Instead of using the class, can you try using a string?

```ruby
@subject.send(rich_text_attribute).instance_of?('ActionText::RichText')
```

Let's see if this change will fix the problem on CI. I almost sure. I'll make some tests.

Reply:
class name evaluation with string poses an issue of being failed silently when the class is not even defined.
But this current scenario is not one of those cases so we may revert to string class comparison.

Reply:
I did not know that. Thank you!